automated/android/apk-automation/apk-automation.yaml

```diff
@@ -45,7 +42,6 @@ run:
         # Upload test output to artifactorial.
         - cp "./output/${TEST_NAME}/result.txt" "./output/result.txt"
         - tar caf "output-${TEST_NAME}.tar.xz" "./output"
-        - if [ -n "${SQUAD_UPLOAD_URL}" ]; then ../../utils/upload-to-squad.sh -a "output-${TEST_NAME}.tar.xz" -u "${SQUAD_UPLOAD_URL}"; fi
-        - if [ -z "${SQUAD_UPLOAD_URL}" ]; then ../../utils/upload-to-artifactorial.sh -a "output-${TEST_NAME}.tar.xz" -u "${ARTIFACTORIAL_URL}" -t "${ARTIFACTORIAL_TOKEN}"; fi
+        - ../../utils/upload-artifacts.sh -a "output-${TEST_NAME}.tar.xz" -u "${UPLOAD_URL}"
```

Comment:
to support my concern, there could be done like this:
```
- if [ -n "${UPLOAD_URL}" ]; ../../utils/upload-artifacts.sh -a "output-${TEST_NAME}.tar.xz" -u "${UPLOAD_URL}"; fi
- if [ -n "${SQUAD_UPLOAD_URL}" ]; ../../utils/upload-artifacts.sh -a "output-${TEST_NAME}.tar.xz" -u "${SQUAD_UPLOAD_URL}"; fi
```
And after like several weeks or a month, drop the SQUAD_UPLOAD_URL line when no SQUAD_UPLOAD_URL jobs will be run.

Reply:
make sense, to keep it around for a while. Good catch.